README.md

```diff
@@ -36,6 +36,18 @@ wget https://raw.githubusercontent.com/k8snetworkplumbingwg/kubemacpool/master/c
 kubectl apply -f ./kubemacpool.yaml
 ``` 
 
+### Opting-in to kubemacpool service
+On releases v0.8.4 and above, kubemacpool is set to apply on pods/vms that reside only on opted-in namespaces. You can opt-in your namespace by adding the following labels:
+- `mutatepods.kubemacpool.io=allocateForAll` - to opt in pods mac allocation in your namespace
+- `mutatevirtualmachines.kubemacpool.io=allocateForAll` - to opt in vms mac allocation in your namespace
+#### Opt-in Example:
+```bash
+[root@hades01 kubemacpool]# kubectl get namespaces --show-labels
```

Comment:
What is more interesting is how to set the label on a namespace via `kubectl`. Then you can list all the namespaces.